css/style.css

```diff
@@ -0,0 +1,2932 @@
+.middle__first {
+  padding-top: 5px;
+  padding-left: 0px;
+}
+
+.middle__first-big {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-right: 7px;
+}
+
+.middle__first-small {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding-left: 0;
+}
+
+.middle__first-title{
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 18px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-top: 8px;
+}
+
+.middle__first-text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+  padding-bottom: 10px;
+}
+
+.middle__text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+}
+
+.middle__btn {
+  margin-top: 95px;
+  margin-left: 12px;
+}
+
+.button__btn {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  border: 2px solid #E51937;;
+  color: #FFFFFF;
+  padding-left: 35px;
+  padding-right: 35px;
+  margin-bottom: 0;
+  margin-right: 10px;
+  background-color: #E51937;
+  margin-left: 13px;
+  margin-top: 148px;
+}
+
+.button__btn-img {
+  padding-right: 7px;
+}
+
+.button__btn-word {
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 13px;
+  padding-right: 0;
+  margin-right: 0;
+  width: 125px;
+}
+
+.button__btn-text {
+  padding: 10px 28px;
+  border: 2px solid #ffffff;
+  font-size: 13px;
+  text-transform: uppercase;
+  color: #ffffff;
+  background-color: #E51937;
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 13px;
+  padding-left: 40px;
+  padding-right: 40px;
+  margin-left: 33px;
+  margin-right: 3px;
+}
+
+.bottom {
+  background: url(../img/model-s-bottom.png) no-repeat;
+  background-position: 647px 142px;
+}
+
+.bottom__inner {
+  display: flex;
+}
+
+.bottom__info {
+  display: flex;
+  flex-direction: column;
+  width: 440px;
+  height: 900px;
+  color: #ffffff;
+  background: #E51937;
+}
+
+.bottom__info-img {
+  position: relative;
+  left: 345px;
+  top: -673px;
+  width: 100%;
+}
+
+.stars__text {
+  font-weight: 600;
+  font-size: 17px;
+ line-height: 21px;
+}
+
+.bottom__description {
+  padding-left: 56px;
+  padding-top: 48px;
+  margin-right: 60px;
+}
+
+.bottom__description-title {
+  width: 320px;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  margin-top: 3px;
+  margin-bottom: 0;
+}
+
+.bottom__description-text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  padding-top: 13px;
+  margin-top: 0;
+  width: 330px;
+}
+
+.bottom__appraisal-title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  padding-left: 56px;
+  padding-top: 41px;
+  margin-top: 0;
+  margin-bottom: 13px;
+}
+
+.bottom__content {
+  padding-top: 0;
+  padding-left: 56px;
+}
+
+.bottom__appraisal-star {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-top: 10px;
+}
+
+.stars {
+ padding-right: 92px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)